src/files/FilesPage.js

```diff
@@ -233,14 +236,19 @@ class FilesPage extends React.Component {
           onInspect={() => this.onInspect(contextMenu.file.hash)}
           onDownload={() => this.onDownload([contextMenu.file])}
           onPin={() => this.props.doFilesPin(contextMenu.file.hash)}
-          onUnpin={() => this.props.doFilesUnpin(contextMenu.file.hash)} />
+          onUnpin={() => this.props.doFilesUnpin(contextMenu.file.hash)}
+          isCliTutorModeEnabled={isCliTutorModeEnabled} // @TODO connect ContextMenu to redux
```

Comment:
Todos are actually done with `//TODO: `, and isn't it connected already?

Reply:
contextMenu page actually isn't connected but also it doesn't need to. Maybe if we'd want, we do can connect `Dropdown` component to reduce prop chaining